test/e2e/framework/nodes.go

```diff
@@ -59,6 +65,31 @@ func findNodesByGatewayLabel(cluster int, isGateway bool) []*v1.Node {
 	return retNodes
 }
 
+// FindMultipleGatewaysCluster finds the cluster with multiple GW nodes.
+// Returns cluster index and active GW Pod and Node names.
+func (f *Framework) FindMultipleGatewaysCluster(clusters []string) (int, map[string]string) {
```

Comment:
@tpantelis 

I just would like to make sure I understand you correctly.
You're saying that instead of passing the `framework.TestContext.ClusterIDs` to the function, better to loop over the `framework.TestContext.ClusterIDs` within the cluster.
And the `framework.TestContext.ClusterIDs` is the globally defined clusters.

Am I get it right?

Reply:
yes just loop over the global `framework.TestContext.ClusterIDs`.

Reply:
Done.